src/components/new-request/name-check/resources/name-check-base-items-structure.ts

```diff
@@ -0,0 +1,316 @@
+import { NameCheckErrorType, NameCheckItemType } from '@/enums'
+
+export const baseItemsStructure = {
+  'conflictsConditional': {
+    icon: 'mdi-alert',
+    iconColor: 'orange darken-2',
+    problem: 'You may need consent to use the following words:',
+    words: [],
+    expandedInfo1: 'Names that include well-known Names ' +
+    '(trade name and Trademarks), names of public figures, or reference ' +
```

Comment:
I think our code style is to indent n-th lines. Is the linter OK with this?

Reply:
linter likes it. I'd rather not go through and tab them all since there's so many and probably won't be changed / looked at much in the future

Reply:
OK, nvm